tests/phpunit/data/themedir1/block-theme/parts/small-header.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:paragraph -->
+<p>Small Header Template Part</p>
+<!-- /wp:paragraph -->
```

Comment:
It seemed easiest to add a `small-header.html` template part since it's already present in the text fixture theme's `theme.json` (for some other tests): https://github.com/WordPress/wordpress-develop/blob/0e478d258b70a94cd2e77b77d00e6b00f2836daa/tests/phpunit/data/themedir1/block-theme/theme.json#L64-L70